### PR TITLE
filter out prerelease versions

### PR DESCRIPTION
**Pull request by alexcb**
Introduces a new prerelease input, which defaults to "false"; prereleased versions will only ever be installed when set to "true".

Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

**Comment by alexcb** [comment]
technically the test was from e9907d2251d22b3bd8b4ca04f31a95a4b7075733 (prior to changing out the github token that's configured)

**Comment by alexcb** [comment]
This also pulls in #15

**Review by nelsam — APPROVED** [review]
lgtm. I think the trailing comma thing is probably fine.

File: src/lib/get-version.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ type ReleaseResponse =
   Endpoints["GET /repos/{owner}/{repo}/releases"]["response"]["data"][0];
 
 export async function getVersionObject(
-  range: string
+  range: string,
+  prerelease: boolean,
```
[the remark this answers is not in the document]
Comment (alexcb) [reply src/lib/get-version.ts:12]: good eyes, fortunately typescript can handle both cases :)

**Pull request merged**